dvc/remote/gdrive.py

```diff
@@ -21,21 +21,27 @@
 
 
 class GDrivePathNotFound(DvcException):
-    def __init__(self, path_info):
-        super().__init__("Google Drive path '{}' not found.".format(path_info))
+    def __init__(self, path_info, hint):
+        hint = "" if hint is None else " {}".format(hint)
+        super().__init__(
+            "GDrive path '{}' not found.{}".format(path_info, hint)
```

Comment:
```suggestion
            "GDrive path '{}' not found. {}".format(path_info, hint)
```
or `\n`?

Reply:
if hint is absent (None) we don't want to see an extra space, see a line above please